config/default.yml

```diff
@@ -30,6 +30,10 @@ Metrics:
 
 Metrics/BlockLength:
   Max: 25
+  Exclude:
+    - spec/
+    - app/serializers/
```

Comment:
This should be in `config/rails.yml` config file.

Reply:
I agree, but `rails.yml` seems to override `default.yml` on this exlusion list